draft-ietf-dprive-xfr-over-tls.md

```diff
@@ -598,12 +855,15 @@ TBD
 
 This document specifies a security measure against a DNS risk: the risk that an
 attacker collects entire DNS zones through eavesdropping on clear text DNS zone
-transfers. It presents a new Security Consideration for DNS. Some questions to
-discuss are: 
+transfers. 
 
-* How should padding be used in IXFR?
-* Should there be an option to 'pad' an AXFR response (i.e. a set of AXFR
-  responses on a given connection) to hide the zone size?
+This does not mitigate:
+
+* the risk that some level of zone activity might be inferred by observing zone transfer sizes and timing on encrypted connections (even with padding applied), in combination with obtaining SOA records by directly querying authoritative servers.
+* the risk that hidden primaries might be inferred or identified via observation of encrypted connections.
+* the risk of zone contents being obtained via zone enumeration techniques.
```

Comment:
Do we think it's worthwhile here to point to the DNS encrypted traffic analysis research by Carmela T et al?

Reply:
Perhaps but I think all the work done to date is for stub to recursive traffic?